fastlane/Fastfile

```diff
@@ -90,6 +90,7 @@ platform :android do
 
   lane :complete_code_freeze do | options |
     android_completecodefreeze_prechecks(options)
+    push_to_git_remote()
```

Comment:
```suggestion
```

This shouldn't be necessary either since `android_completecodefreeze_prechecks` is not making any changes to the project.